### PR TITLE
Yautja fluff content update + hugger tank update

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Adds gigantic amount of assets for decorations, items, machinery etc for yautja

Mostly a port of things that i was working on from https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps

Also contains a port of https://github.com/tgstation/TerraGov-Marine-Corps/pull/12776
Which adds a living huggers and dead larva in destroyed tanks

Almost all new sprites by medjic (discord tag)

Skull spites by 3raz3rr (discord tag)

Statue sprites by crowford

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

StrongDMM_tbf3jvtCfT

StrongDMM_ezBUFFT9Gn

dreamseeker_VWcT6Ci6Wy

dreamseeker_zHRBACuntx

dreamseeker_TDZMDY5WPX

dreamseeker_nKXFRO5MKF

dreamseeker_Y5xZZ5ix6n

</details>

# Changelog
:cl: Blundir, medjic, 3raz3rr, Crowford
add: hugger and larva tanks now actually have an alive hugger and a dead larva inside them on destroy
add: yautja: beds, chairs, microwaves, soda/chem/booze dispensers, sleeper/scanner/autodoc, freezer, refrigerator, air tank, computer, vendors, gibber, food grinder, hugger tank, closet, water/fuel tanks, juicer assets and objects
add: unique yautja full set of normal tools, replaced tools in yautja toolbelt with them
add: yautja windows 
add: yautja new statues
add: decorative xeno skulls for yautja locations
fix: preds not being able to use chem master and chem dispenser on their ship
fix: preds not having access to vendors on their ship
del: unused yautja ship map
mapadd: added tons of new assets on LV624, New Varadero, Trijent Dam, Yautja ship
/:cl:

### CODE REVIEW

**Review by nauticall — CHANGES_REQUESTED** [review]
image
only real suggestion (sort of) is to darken the sprite on the backrests of these chairs to provide better contrast -- they look a bit flat otherwise without it. rest is good

**Review by nauticall — APPROVED** [review]
it is what it is

File: code/game/objects/structures/misc.dm [hunk 1]
```diff
@@ -106,31 +115,115 @@
...
+		return FALSE
+	health = max(0, health - damage)
+
+	if(health == 0)
+		to_chat(loc, SPAN_DANGER("[src] shatters!"))
+		deconstruct(FALSE)
+		playsound(src, "shatter", 25, 1)
+		return TRUE
+
+	return FALSE
```
Comment (kiVts) [inline code/game/objects/structures/misc.dm:156]: signle letter vars

File: code/game/objects/structures/misc.dm [hunk 2]
```diff
@@ -106,31 +115,115 @@
...
+	playsound(user.loc, 'sound/effects/Glasshit.ogg', 25, 1)
+	take_damage(W.demolition_mod*W.force)
+
+/obj/structure/xenoautopsy/tank/proc/take_damage(damage, mob/M)
+	if(!damage)
+		return FALSE
+	health = max(0, health - damage)
+
+	if(health == 0)
+		to_chat(loc, SPAN_DANGER("[src] shatters!"))
```
Comment (kiVts) [inline code/game/objects/structures/misc.dm:151]: visisble_message()

File: code/modules/cm_preds/yaut_items.dm [hunk 3]
```diff
@@ -1242,3 +1242,132 @@
...
+	item_state = "multitool"
+	item_icons = list(
+		WEAR_L_HAND = 'icons/mob/humans/onmob/hunter/items_lefthand.dmi',
+		WEAR_R_HAND = 'icons/mob/humans/onmob/hunter/items_righthand.dmi'
+	)
+
+/obj/item/tool/weldingtool/yautja
+	name = "alien chem welding tool"
+	desc = "A complex chemical welding device, keep away from youngblood."
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
```
Comment (kiVts) [inline code/modules/cm_preds/yaut_items.dm:1349]: \improper for these

**Review by kiVts — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/misc.dm [hunk 4]
```diff
@@ -106,31 +115,115 @@
...
+		return TRUE
+
+	return FALSE
+
+/obj/structure/xenoautopsy/tank/bullet_act(obj/projectile/Proj)
+	bullet_ping(Proj)
+	if(Proj.ammo.damage)
+		take_damage(floor(Proj.ammo.damage / 2))
+		if(Proj.ammo.damage_type == BRUTE)
+			playsound(src.loc, 'sound/effects/Glasshit.ogg', 25, 1)
```
Comment (kiVts) [inline code/game/objects/structures/misc.dm:163]:
```suggestion
		return TRUE

	return FALSE

/obj/structure/xenoautopsy/tank/bullet_act(obj/projectile/Proj)
	bullet_ping(Proj)
	if(Proj.ammo.damage)
		take_damage(floor(Proj.ammo.damage / 2))
		if(Proj.ammo.damage_type == BRUTE)
			playsound(loc, 'sound/effects/Glasshit.ogg', 25, 1)
```

**Review by kiVts — APPROVED** [review]

**Review by kiVts — APPROVED** [review]
code^

**Review by Nanu308 — APPROVED** [review]
mapping approved, old ship deleted.
Overall improvement to detailing and more custom pred sprites are good.

**Pull request merged**